pages/[[...slug]].tsx

```diff
@@ -277,12 +277,10 @@ export default function Home({ onTouchReady }) {
       <header className={styles.nav}>
         <Dialog open={aboutOpen} onOpenChange={setAboutOpen}>
           <DialogTrigger asChild>
-            <button style={{ all: "unset" }}>
-              <div className={styles.logo}>
```

Comment:
This inner wrapper seems to be not needed. It was required a few years ago to make flexbox work in `<button>` elements in some browsers (e.g. Firefox), but not anymore.